src/Sylius/Component/User/Model/User.php

```diff
@@ -0,0 +1,670 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This component was inspired by FOS User-Bundle
+ */
+
+namespace Sylius\Component\User\Model;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
+ */
+class User implements UserInterface
+{
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var CustomerInterface
+     */
+    protected $customer;
+
+    /**
+     * @var string
+     */
+    protected $username;
```

Comment:
So it's User.username <=> Customer.email ?

What about situations where the username is not the email? (This doesn't apply to us, we do have emails as usernames), but is this enforcing a rule here that it maybe shouldn't?